pkgs/development/libraries/mesa/default.nix

```diff
@@ -37,12 +39,12 @@ stdenv.mkDerivation {
 
   src = fetchurl {
     urls = [
+      "https://mesa.freedesktop.org/archive/mesa-${version}.tar.xz"
```

Comment:
Is it ok to move the HTTPS link to the top? That would make `nix-prefetch-url -A mesa.src` more secure, but since Hydra has to build it as well MITM attacks are luckily more difficult anyway.

Reply:
The insecurity happens only when someone is changing the sha256 written here.  But I think it's a good idea to do the move anyway.